docs/installation/other_inst_scenarios/rke2.tape

```diff
@@ -0,0 +1,71 @@
+Output ../../../static/img/rke2.gif
+
+Set FontSize 14
+Set Width 1100
+Set Height 700
+Set WindowBar Rings
+Set Padding 5
+Set Margin 5
+Set MarginFill "#30BA78"
+Set TypingSpeed 50ms
+
+Hide
+Type "ssh -X 192.168.122.109"
+Enter
+Sleep 2
+Type "sudo bash"
+Enter
+Type "cd"
+Enter
+Show
+Sleep 1s
+
+Type "curl -sfL https://get.rke2.io | sh -"
+Sleep 500ms
+Enter
+Sleep 10s
+Type "systemctl enable --now rke2-server.service"
+Sleep 500ms
+Enter
+Sleep 50s
```

Comment:
This is a lot of time waiting for anything, with no visible response at all.
Can we `Type "# ... skipping waiting for a minute"` here and reduce the sleep to `500ms` ?
Problem with that is that this file looks to be an instruction stream driving a terminal.
The command responses are not shown, and will appear only in the result, I suspect.

Reply:
I'm sure something better is possible. It's just the (my) first iteration with it.